setup.sh

```diff
@@ -0,0 +1,382 @@
+  while [[ "$setVmParam" != [yYnN] ]]
+  do
+    promptBold "[❓] Do you want to set the vm.max_map_count now? (y/N) "
+    read setVmParam trash
+    case "$setVmParam" in
+      [yY])
+        echo
+        writeBold "Setting kernel variable vm.max_map_count to $REQUIRED_MAP_COUNT..."
+        sysctl -w vm.max_map_count=$REQUIRED_MAP_COUNT
+        if [ -z "$MAP_COUNT" ]; then
+          echo "vm.max_map_count=$REQUIRED_MAP_COUNT" > $SYSCTL_CONF_FILE
+        else
+          sed 's/vm.max_map_count=.+/vm.max_map_count=$REQUIRED_MAP_COUNT/g' $SYSCTL_CONF_FILE > $SYSCTL_CONF_FILE
+        fi
+        echo
+        writeBold "$GREEN" "[✔] Kernel variable successfully set."
+        ;;
+      [nN] | '')
+        echo
+        writeBold "$BLUE" "Ok. Please set the kernel variable and re-run this script."
+        echo
+        exit 0
+        ;;
+      *)
+      echo
+      writeBold "$RED" "[✖] I did not understand your answer."
+      ;;
+    esac
+  done
+}
+
+collectPersonalData() {
+  echo
+  writeBold  "[ℹ] Please let us know a little bit about youself."
+  promptBold "    What's your email address?${NORMAL} (press Enter to skip)"
+  read email trash
+  promptBold "    What's your name?${NORMAL} (press Enter to skip)"
+  read firstName lastName otherName yetAnotherName trash
+  promptBold "    What do you plan to use Kuzzle for?${NORMAL} (press Enter to skip)"
+  read purpose
+  # TODO send collected data to analytics service
+}
+
+startKuzzle() {
+  composerYMLURL="https://raw.githubusercontent.com/kuzzleio/kuzzle-build/master/docker-compose/kuzzle-docker-compose.yml"
+  composerYMLPath="kuzzle-docker-compose.yml"
+  if commandExists curl; then
+    echo
+    writeBold "Downloading Kuzzle launch file..."
+    echo
+    curl -XGET $composerYMLURL > $composerYMLPath
+  elif commandExists wget; then
+    echo
+    writeBold "Downloading Kuzzle launch file..."
+    echo
+    wget -O $composerYMLPath $composerYMLURL
+  else
+    writeBold "$RED" "[✖] curl or wget need to be installed to download the Kuzzle launch file,"
+    writeBold "$RED" "    but none seems to be installed on your system."
+    echo
+    writeBold "$BLUE" "Please install curl or wget and re-run this script."
+    echo
+    exit 3
+  fi
+  echo
+  writeBold "$GREEN" "[✔] The Kuzzle launch file has been successfully downloaded."
+  writeBold          "    This script can launch Kuzzle automatically or you can do it"
+  writeBold          "    manyally using Docker Compose."
+  write              "    To manyally launch Kuzzle you can type the following command:"
+  write              "    docker-compose -f $composerYMLPath up"
+  while [[ "$launchTheStack" != [yYnN] ]]
+    do
+      promptBold "[❓] Do you want to start Kuzzle now? (y/N) "
+      read launchTheStack trash
+      case "$launchTheStack" in
+        [yY])
+          echo
+          writeBold "Starting Kuzzle..."
+          $DOCKER_COMPOSE_BIN -f $composerYMLPath up -d
```

Comment:
To me, `$DOCKER_COMPOSE_BIN` is the default install dir of docker-compose. That's how it is used until the current script line anyway.

Here, I would use `$(command -v docker-compose)` instead, as docker-compose might already be installed... but not in the directory you set at the beginning of this script.